task_schedule.pl

```diff
@@ -68,13 +68,19 @@
 
 help(2) if ($opt{help});
 
+# If config <path> is not an absolute path then look for config
+# in $SKA_ARCH_OS/share/<path>.  This is the preferred convention
+# for skare3 packages that are fully installable within arch.
+if (not $opt{config} =~ m|^/|) {
+    $opt{config} = "$ENV{SKA_ARCH_OS}/share/$opt{config}";
```

Comment:
So for this, $ENV{SKA_ARCH_OS} is set to $SKA/arch/x86_64-linux_CentOS-5/ by CXC::Envs::Flight above.  So, to work with our new no-arch install do we want to redo CXC::Envs::Flight to set SKA_ARCH_OS to SKA, or do we just want to change this line to be $ENV{SKA}/share/$opt{config} ?